src/main/java/com/autotune/operator/KruizeDeploymentInfo.java

```diff
@@ -65,6 +65,14 @@ public class KruizeDeploymentInfo {
     public static String database_admin_username;
     public static String database_admin_password;
     public static String database_ssl_mode;
+
+    public static String cloudwatch_logs_access_key_id;
+    public static String cloudwatch_logs_secret_access_key;
+    public static String cloudwatch_logs_log_group;
+    public static String cloudwatch_logs_region;
+    public static String cloudwatch_logs_log_level;
+    public static String cloudwatch_logs_log_stream;
```

Comment:
Can you also please make this variable set via env 
you can do that by 
 declaring  it in KruizeConstants.KRUIZE_CONFIG_ENV_NAME
 
 public static final String CLOUDWATCH_LOGS_ACCESS_KEY_ID = "cloudwatch_accesskeyid";  
 make sure to have only one Underscore in value 


and in crc yaml in ConfigMap -> in kruizeconfigjson

declare like this
"cloudwatch" : {
         "accesskeyid" : "",
         "secretaccesskey:"",   etc..
}

Reply:
@kusumachalasani - Can you please document the steps to setup and test this

Reply:
Updated the kruizeconfig related design documents for the same.
To enable cloudwatch logs, we need to provide aws access / secret keys as mentioned in the document.

Reply:
@kusumachalasani - I don't see the doc changes in this PR